### DISCUSSION TITLE
No "Share" Button to create sharable links

### DISCUSSION THREAD

**Discussion by CoreyMcCoy** [vercel/community#6709] [Help] [answered]
### Summary

The docs and AI support assitant refer to a "Share" button on the deployment page, but there is no button. How do I create a sharable link that can be protected now?

### Steps to Reproduce

Followed the directions from documentation and customer support AI

**Comment by swarnava** [comment]
Hi,

Can you ensure if you have Vercel Authentication enabled under Settings > Deployment Protection ?

**Reply by CoreyMcCoy** [reply]
![proof](https://github.com/vercel/community/assets/19680931/ce21b823-171e-4e67-aabb-ffb1c84bd84f)

**Comment by CoreyMcCoy** [comment]
I'm on a hobby plan. Per the docs I can set up one sharable link.

Please see attached image.

Corey McCoy

**Comment by CoreyMcCoy** [comment]
Hello?? What's the deal?

On Thu, May 2, 2024 at 4:22 AM Swarnava Sengupta ***@***.***>
wrote:

> Hi,
>
> Can you ensure you have Pro plan and Vercel Auth is enabled? Otherwise
> your deployment is publicly available.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/vercel/discussions/6709#discussioncomment-9293410>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AEWE5I42BOCJYYT62AAT4RLZAIAW5AVCNFSM6AAAAABHDDC43GVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TEOJTGQYTA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Corey McCoy

**Reply by swarnava** [reply]
Can you share the deployment url?

**Reply by CoreyMcCoy** [reply]
Here you go.
https://naturalization-quiz-c4jfmjwvz-coreys-projects-33fcbb71.vercel.app/

**Reply by swarnava** [reply, marked as answer]
This looks like a Production deployment and already available to public. You can enable this for preview deployment. [Deployment url](https://naturalization-quiz-c4jfmjwvz-coreys-projects-33fcbb71.vercel.app/) can't be shared beyond the team
